python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```
@@ -1560,6 +1560,9 @@ $defs:
                     description: |
                       EOS cli commands rendered on router_bgp.vlans.
                       This configuration will not be applied to vlan aware bundles.
+              private_vlan:
+                type: dict
+                $ref: eos_cli_config_gen#/keys/vlans/items/keys/private_vlan
```

maybe we should make the keys required here. Keeping $ref so we inherit the rest. Doing this may require a code change to properly convert the types.
```suggestion
                $ref: eos_cli_config_gen#/keys/vlans/items/keys/private_vlan
                keys:
                  type:
                    type: str
                    required: true
                  primary_vlan:
                    type: ind
                    required: true
```